apps/web/src/utils/poolCandidate.ts

```diff
@@ -80,7 +80,11 @@ export const isSuspendedStatus = (
 ): boolean => {
   const isSuspended = suspendedAt && new Date() > parseDateTimeUtc(suspendedAt);
 
-  return !!(isSuspended && status === PoolCandidateStatus.QualifiedAvailable);
+  return !!(
+    isSuspended &&
+    (status === PoolCandidateStatus.QualifiedAvailable ||
+      status === PoolCandidateStatus.PlacedCasual)
```

Comment:
Same comment about naming this to make it easier to understand.  I think that "QualifiedAvailable and PlacedCasual" should probably be one of the categories in constants/poolCandidate, or a new category should get created.  SUSPENDABLE_STATUSES?

Reply:
I think a category should be created when we've got more than one applicable entity for some purpose. Ease of consistency and reuse that way
https://github.com/GCTC-NTGC/gc-digital-talent/pull/12741/commits/a8e426ac7baf22ec1683662a5971e014b62dc172